packages/medusa/src/loaders/helpers/get-model-extension-map.ts

```diff
@@ -0,0 +1,57 @@
+import glob from "glob"
+import path from "path"
+import { EntitySchema } from "typeorm"
+
+import { formatRegistrationName } from "../../utils/format-registration-name"
+import { ClassConstructor } from "../../types/global"
+
+type GetModelExtensionMapParams = {
+  directory?: string
+  pathGlob?: string
+  config: Record<string, any>
+}
+
+export function getModelExtensionsMap({
+  directory,
+  pathGlob,
+  config = {},
+}: GetModelExtensionMapParams): Map<
+  string,
+  ClassConstructor<unknown> | EntitySchema
+> {
+  const modelExtensionsMap = new Map<
+    string,
+    ClassConstructor<unknown> | EntitySchema
+  >()
+  const fullPathGlob =
+    directory && pathGlob ? path.join(directory, pathGlob) : null
+
+  const modelExtensions = fullPathGlob
+    ? glob.sync(fullPathGlob, {
+        ignore: ["index.js"],
```

Comment:
Should we ignore index.map.js if the user build with the source map?

Reply:
done all